helloworld-rs/pom.xml

```diff
@@ -58,14 +78,84 @@
             <scope>provided</scope>
         </dependency>
 
-        <!-- Import the JAX-RS API, we use provided scope as the API is included in JBoss EAP -->
         <dependency>
-            <groupId>org.jboss.spec.javax.ws.rs</groupId>
-            <artifactId>jboss-jaxrs-api_2.1_spec</artifactId>
-            <scope>provided</scope>
+          <groupId>io.quarkus</groupId>
+          <artifactId>quarkus-resteasy</artifactId>
```

Comment:
Replace JAX-RS dependency

Reply:
should use quarkus-resteasy-reactive if possible. 

might be edge cases of things in resteasy/jaxrs we don't support but quarkus-resteas-reactive would be recommended if intent is to move forward vs staying more compatible.